samples/CGCatalog/CGCatalog/Samples/CGCCGContextReplacePathWithStrokedPath.m

```diff
@@ -50,36 +57,86 @@ - (void)drawRect:(CGRect)rect {
     NSMutableArray<NSValue*>* originalPoints = [NSMutableArray new];
     CGMutablePathRef path = CGPathCreateMutable();
     CGContextSetLineWidth(context, c_LineWidth);
-    CGPathMoveToPoint(path, NULL, 10, maxHeight / 2);
-    CGPathAddLineToPoint(path, NULL, maxWidth / 4, maxHeight / 2);
-    CGPathAddArcToPoint(path, NULL, maxWidth / 4 + 50, maxHeight * 2 + 50, maxWidth / 2, maxHeight * 3 / 4, 50);
-    CGPathAddQuadCurveToPoint(path, NULL, maxWidth * 3 / 4, maxHeight * 1 / 4, maxWidth - 10, 10);
+    CGContextSetLineCap(context, kCGLineCapRound);
+    CGContextSetLineJoin(context, kCGLineJoinMiter);
+    CGContextSetMiterLimit(context, 1);
+
+    if (self.strokedPathType == StrokedPathTypeReplaceStroke2) {
+        CGPathMoveToPoint(path, NULL, maxWidth / 2, maxHeight / 4);
+        CGPathAddLineToPoint(path, NULL, 16, 16);
+    } else {
+        CGPathMoveToPoint(path, NULL, 50, 50);
+        CGPathAddLineToPoint(path, NULL, 100, 100);
+    }
+
+    if (self.strokedPathType == StrokedPathTypeReplaceStroke2) {
+        CGPathMoveToPoint(path, NULL, maxWidth - 16, 16);
+        CGPathAddQuadCurveToPoint(path, NULL, maxWidth * 3 / 4, maxHeight * 1 / 4, maxWidth / 2, maxHeight * 3 / 4);
+        CGPathAddArcToPoint(path, NULL, maxWidth / 2, maxHeight * 1.25, 0, maxHeight / 4, 50);
+        CGPathAddLineToPoint(path, NULL, maxWidth / 4, maxHeight / 2);
+        CGPathAddLineToPoint(path, NULL, 10, maxHeight / 2);
+    } else {
+        CGPathMoveToPoint(path, NULL, 200, 200);
+        CGPathAddLineToPoint(path, NULL, 260, 100);
+        CGPathAddArcToPoint(path, NULL, 500, 500, 450, 450, 50);
+        CGPathAddQuadCurveToPoint(path, NULL, 100, 200, 200, 300);
+    }
+
```

Comment:
Issues #630 => the points actually don't show.  Is the sample behavior tested on winobjc?